junit/src/main/java/cucumber/runtime/junit/JUnitReporter.java

```diff
@@ -202,7 +203,7 @@ public void fireTestFinished() {
         public void addFailure(Throwable targetException) {
             if (targetException instanceof MultipleFailureException) {
                 addMultipleFailureException((MultipleFailureException) targetException);
-            } else if (isPending(targetException)) {
+            } else if (isAssumptionViolated(targetException)) {
```

Comment:
Do you intend to count the pending exception as a failure? I'm not entirely sure how pending items should be reported? I guess this should depend on the strict setting.